syz-manager/manager.go

```diff
@@ -100,7 +100,7 @@ type Manager struct {
 
 	mu                    sync.Mutex
 	fuzzer                atomic.Pointer[fuzzer.Fuzzer]
-	source                queue.Source
+	source                *queue.Distributor
```

Comment:
Nit: maybe rename it to something like `snapshotSource` to make it less confusing? At first glance it looks as if it's also something related to the RPCServer.

Reply:
Done.